Formula/l/llvm.rb

```diff
@@ -122,7 +126,7 @@ def install
       -DLLDB_ENABLE_LUA=OFF
       -DLLDB_ENABLE_LZMA=ON
       -DLLDB_PYTHON_RELATIVE_PATH=libexec/#{site_packages}
-      -DLIBOMP_INSTALL_ALIASES=OFF
+      -DLIBOMP_INSTALL_ALIASES=ON
```

Comment:
I don't think this is a good idea -- this is disabled by default because these aliases shadow the GCC's OpenMP libraries.